ui/app/mainui/AppMain.qml

```diff
@@ -224,6 +224,31 @@ Item {
                                            "%1/%2".arg(appMain.rootStore.getEtherscanLink(chainId)).arg(txHash))
             }
         }
+
```

Comment:
It would be better to use, as I explained before, the `ToastsManager.qml` file for that!